docs/core/tools/dotnet-environment-variables.md

```diff
@@ -156,17 +156,23 @@ When <xref:System.Console.IsOutputRedirected?displayProperty=nameWithType> is `t
 
 ### `DOTNET_DiagnosticPorts`
 
-Configures .NET processes to connect to [dotnet monitor](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration.md#connection-mode) when dotnet monitor is in Listen mode.
+Configures alternate endpoints where diagnostic tools can communicate with the .NET runtime. See the [Diagnostic Port documentation](../diagnostics/diagnostic-port.md) for more information.
 
-### Mono-specific variables
+### `DOTNET_DefaultDiagnosticPortSuspend`
 
-- `DOTNET_DefaultDiagnosticPortSuspend`: Configures the runtime to pause during startup and wait for the _Diagnostics IPC ResumeStartup_ command from the specified diagnostic port.
-- `DOTNET_EnableDiagnostics`: When set to `1`, enables Mono diagnostics.
-- `DOTNET_EnableEventPipe`: When set to `1`, enables the Mono event pipe.
-- `DOTNET_EventPipeOutputPath`: The output path for the Mono event pipe.
-- `DOTNET_EventPipeOutputStreaming`: When set to `1`, enables Mono event pipe output streaming.
+Configures the runtime to pause during startup and wait for the _Diagnostics IPC ResumeStartup_ command from the specified diagnostic port when set to 1. Defaults to 0. See the [Diagnostic Port documentation](../diagnostics/diagnostic-port.md) for more information.
 
-For more information, see [.NET runtime: Mono diagnostics and tracing](https://github.com/dotnet/runtime/blob/main/docs/design/mono/diagnostics-tracing.md).
+### `DOTNET_EnableDiagnostics`
+
+When set to `1`, enables debugging, profiling, and other diagnostics via the [Diagnostic Port](../diagnostics/diagnostic-port.md). Defaults to 1.
```

Comment:
Doesn't setting this value to `0` turn off more than the diagnostics server? I thought it disabled debugging as well.

Reply:
Yes setting it to 0 does disable debugging and profiling as well. Does the wording not make it sound that way?

Reply:
The phrasing right now implies to me that it turns off these features only in relation to the diagnostic port rather than entirely. I think it's the "and other diagnostics via the diagnostic port" part that makes the first two items in the list sound dependent on the diagnostic port. It's not a huge deal since I'm probably just nitpicking. I'm okay if you leave it as is.